flask-server/main.py

```diff
@@ -284,10 +284,15 @@ def add_clink():
             }
             return response_object, 401
 
+        private = False
+        if request.json['privacy']:
+            private = True    
```

Comment:
are you able to set private = request.json['privacy']

Reply:
the only thing that complicate that is that i think values.privacy is set to false but rather is null if unchecked, but i will try and get back to you

Reply:
yeah if you console.log(values), values.privacy is undefined not false when unchecked

Reply:
I think it might be cuz you didn't explicitly set privacy to be false when you make the post request?